content/rdi/quickstart/write-behind-guide.md

```diff
@@ -229,6 +236,106 @@ To use the metrics you can either:
 
 - Scrape the metrics using RDI Prometheus exporter
 
+### Read-through jobs
+
+Read-through jobs are a mandatory part of the read-through pipeline configuration.
+Under the `jobs` directory (parallel to `config.yaml`) you should have a job definition in a YAML file per every key pattern you want to read into cache from a downstream database table in case of cache miss.
+
+The YAML file can be named using the database table name or another naming convention, but has to have a unique name.
```

Comment:
`has to have a unique name` - it's impossible to create the same file name under the same directory anyway, so maybe this part is redundant.

Reply:
done